src/main/java/org/gridsuite/study/server/service/ConsumerService.java

```diff
@@ -58,19 +59,25 @@ public class ConsumerService {
     NotificationService notificationService;
     StudyService studyService;
     CaseService caseService;
+    NetworkStoreService networkStoreService;
```

Comment:
It is unused, I reckon you can remove it

Reply:
Yes removed